.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -5,25 +5,27 @@ about: Create a bug report to help us improve Streamlink
 
 <!--
 Thanks for reporting a bug!
+USE THE TEMPLATE, otherwise, your bug report may be rejected.
```

Comment:
I don't think we need these commas.

Reply:
Before I change this, does
"USE THE TEMPLATE! Otherwise your bug report may be rejected."
sound better? The exclamation mark may sound a bit harsh.

Reply:
I'd avoid the exclamation mark, I agree it sounds harsh.